service/src/main/resources/application.yaml

```diff
@@ -67,6 +67,8 @@ fpl:
     mapping: 'example=>cafcass:cafcass@cafcass.com'
   local_authority_code_to_shared_inbox:
     mapping: 'example=>local-authority@local-authority.com'
+  default_email:
```

Comment:
this is different everywhere. Please standardise.

Reply:
it will be same mapping as `fpl.default_email.mapping` for all the places

Reply:
changed to `local_authority_fallback_inbox`